package.json

```diff
@@ -46,6 +46,7 @@
     "@angular/platform-browser-dynamic": "14.2.1",
     "@angular/router": "14.2.1",
     "@angular/service-worker": "14.2.1",
+    "@bitcoinerlab/secp256k1": "^1.1.1",
```

Comment:
Is this really needed? Please use existing libraries for secp256k1.

Reply:
I was using the existing library , but got some errors. I found this stack article with similar issues:
https://bitcoin.stackexchange.com/questions/118613/webpack-issue-with-tiny-secp256k1-and-ecpair

secp256k1 is suitable replacement, and more widely supported, while also fixed the error.

Reply:
I see the library is bade on noble-library, so I'll approve this for now.